src/plugins/console/public/styles/_app.scss

```diff
@@ -133,10 +133,52 @@
 .console__monaco_editor__selectedRequests {
   background: transparentize($euiColorLightShade, .3);
 }
+
 /*
- * The z-index for the autocomplete suggestions popup
+ * The styling for the multiple-response status code decorations
  */
+%monaco__status_badge {
+  font-family: $euiFontFamily;
+  font-size: $euiFontSizeS;
+  font-weight: $euiFontWeightMedium;
+  line-height: $euiLineHeight;
+  padding: 2px $euiSizeXS;
+  display: inline-block;
+  border-radius: calc($euiBorderRadius / 2);
+  white-space: nowrap;
+  vertical-align: top;
+  cursor: default;
+  max-width: 100%;
+}
+
+.monaco__status_badge__primary {
+  @extend %monaco__status_badge;
+  background-color: $euiColorVis1;
+}
+
+.monaco__status_badge__success {
+  @extend %monaco__status_badge;
+  background-color: $euiColorSuccess;
+}
+
+.monaco__status_badge__default {
+  @extend %monaco__status_badge;
+  background-color: $euiColorLightShade;
+}
+
+.monaco__status_badge__warning {
+  @extend %monaco__status_badge;
+  background-color: $euiColorWarning;
+}
+
+.monaco__status_badge__danger {
+  @extend %monaco__status_badge;
+  background-color: $euiColorDanger;
+}
```

Comment:
nit: it's not clear what naming convention you're using for the class names, but it sort of feels close to BEM, if it is the modifiers should be of the following format `.monaco_status_badge--primary`, see https://getbem.com/naming/

Reply:
Thanks for the review and good call! The names are fixed in [0538534](https://github.com/elastic/kibana/pull/189210/commits/0538534133858e47d7ef32a7674cf42a4d648ec7).